### DISCUSSION TITLE
How to quickly learn to use Git and GitHub?

### DISCUSSION THREAD

**Discussion by yanghm123** [community/community#22587] [New to GitHub]
<p>I am a newbie. Can anyone explain how to quickly learn and use Git and GitHub? Welcome everyone to leave a message below. thank you</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/yanghm123">@yanghm123</a>,</p>
<p>You can find the same question here: <a href="https://github.community/t/who-has-a-good-way-to-learn-github/1495/3">https://github.community/t5/How-to-use-Git-and-GitHub/Who-has-a-good-way-to-learn-GitHub/m-p/14972#M4656</a></p>
<p>I copy and paste the same solution.</p>
<p>There are a lot of way to learn git and in particularly github:</p>
<ul>
<li><a href="https://lab.github.com/" rel="nofollow noopener">https://lab.github.com/</a></li>
<li><a href="https://guides.github.com/" rel="nofollow noopener">https://guides.github.com/</a></li>
<li>if you want a video courses there is udemy</li>
<li>even on youtube and books</li>
</ul>
<p>Tip: you can easily search threads by using the search box, this prevent to create duplicate threads.</p>
<p>-Gabriele-</p>

**Comment by yanghm123** [comment]
<p>OK   I will go see  thank you</p>
